tools/obscuroscan_v3/frontend/README.md

```diff
@@ -0,0 +1,40 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
```

Comment:
Should this file be updated ? Would it make sense to clean up the file and remove stuff like the deploy instructions ?